ffcx/ir/representationutils.py

```diff
@@ -47,18 +46,18 @@ def id(self):
 def create_quadrature_points_and_weights(integral_type, cell, degree, rule):
     """Create quadrature rule and return points and weights."""
 
+    # from IPython import embed; embed()
     if integral_type == "cell":
-        (points, weights) = create_quadrature(cell.cellname(), degree, rule)
+        return create_quadrature(cell.cellname(), degree + 1, rule)
```

Comment:
Why is it `degree+1` compared to `degree`?

Reply:
I don't remember, but it appears that everything still passes if I remove the `+ 1`. Perhaps there was a bug in Basix that we've since fixes